### ISSUE TITLE
The EventBus#once function is not working correctly

### ISSUE THREAD

**Issue by micooz** [quasarframework/quasar#16202]
### What happened?

`EventBus#once` is not working as expected. The listener callback function's parameters is always the same as the first parameter(`event`) of the `EventBus#emit` method, which is not the expected behavior.

### What did you expect to happen?

The listener callback function's parameters of `EventBus#once` should follow the rest parameters passed by `EventBus#emit` method, not the first `event` parameter.

### Reproduction URL

https://codepen.io/micooz/pen/abPoRKd

### How to reproduce?

1. Go to reproduction link
2. Open console panel

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Quasar CLI Commands/Configuration (@quasar/cli | @quasar/app-webpack | @quasar/app-vite)

### Platforms/Browsers

Chrome

### Quasar info output

```Shell
Operating System - Darwin(22.5.0) - darwin/arm64
NodeJs - 18.17.0

Global packages
  NPM - 9.6.7
  yarn - Not installed
  @quasar/cli - 2.2.2
  @quasar/icongenie - Not installed
  cordova - Not installed

Important local packages
  quasar - 2.12.4 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app-vite - 1.4.3 -- Quasar Framework App CLI with Vite
  @quasar/extras - 1.16.5 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.3.4 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.2.4
  pinia - 2.1.4 -- Intuitive, type safe and flexible Store for Vue
  vuex - Not installed
  vite - 2.9.16 -- Native-ESM powered web dev build tool
  eslint - 8.45.0 -- An AST-based pattern checker for JavaScript.
  electron - 25.3.1 -- Build cross platform desktop apps with JavaScript, HTML, and CSS
  electron-packager - 17.1.1 -- Customize and package your Electron app with OS-specific bundles (.app, .exe, etc.) via JS or CLI
  electron-builder - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*
```

**Comment by micooz** [comment]
I opened a pull request to fix this bug, pls review: https://github.com/quasarframework/quasar/pull/16203

**Comment by rstoenescu** [comment]
Thanks for the PR! Changed the implementation a bit and pushed.
Fix will be available in Quasar v2.12.5
